operator/cmd/mesh/install.go

```diff
@@ -185,6 +186,32 @@ func runApplyCmd(cmd *cobra.Command, rootArgs *rootArgs, iArgs *installArgs, log
 		return fmt.Errorf("failed to install manifests: %v", err)
 	}
 
+	// Make this revision the default if none exists
```

Comment:
I don't know how much of this is broken already, but `istioctl x unintsall` leads around all the webhooks (including validation) which ends up breaking

Reply:
Hm yeah it seems like the webhooks get removed when doing `uninstall --purge` but not `uninstall --revision` (even though the `istio.io/rev` label is there)... I'll try and fix as part of this PR.

Reply:
Fixed.